cypress/e2e/swag_lab/Burger_menu_button.cy.js

```diff
@@ -0,0 +1,25 @@
+describe ('Product page', ()=>{
+    it ('product page should be visible', ()=>{
+        cy.visit('https://www.saucedemo.com');
+        cy.get('#user-name').type("standard_user");
+        cy.get('#password').type("secret_sauce");
+        cy.get('#login-button').click();
+
+        cy.url().should('include','/inventory.html');
+        cy.contains('Swag Labs'); 
+
+        // Click on burger menu button 
+        cy.get('#react-burger-menu-btn').should('be.visible').and('be.enabled');
+        cy.get('#react-burger-menu-btn').click();
```

Comment:
line 12 and 13 can be combined and written in a single assertion chain like:
cy.get('#react-burger-menu-btn')
.should('be.visible')
.and('be.enabled')
.click();

this is a standard way of writing code.